res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -858,6 +859,7 @@ NumarkMixtrack3.deck = function(decknum) {
     // NMTP3 is a "Model A" controller for scratching, it centers on 0.
     // See http://www.mixxx.org/wiki/doku.php/midi_scripting#scratching
     // and see "Jogger" object constructor
+    this.beatJumpSize = 1;
```

Comment:
That would be awesome. I will not be committing code that will modify this value until the CO will be there as this controller has no way of displaying it.